<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html lang="en">
<head>
<link rel="stylesheet" href="../../../stylesheets/styles.css">
<link rel="stylesheet" href="../../../stylesheets/pygment_trac.css">
<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">

<meta http-equiv="Content-type" content="text/html; charset=UTF-8">
<meta http-equiv="Content-Style-Type" content="text/css">
<title>ASCI896-004</title>
</head>
<body>
<p class="menublock"><a href="../index.html">Back</a></p>

<h1>ASCI 896-004 Statistical Genomics</h1>
<h2>Spring 2016</h2>


<br>
<br>
<h3>Instructor</h3>
<ul>
  <li>Name: Gota Morota</li>
  <li>Office: A218f Animal Science Building</li>
  <li>Email: morota@unl.edu</li>
  <li>Web: <a href="http://morotalab.org/">http://morotalab.org/</a></li>
  <li>Office Hours: By appointment</li>
</ul>

<br>
<br>
<h3>Time and Location</h3>
<ul>
  <li>Tues./Thurs. 9:30-10:45am</li>
  <li>Animal Science Building, Room A228</li>
</ul>

<br>
<br>
<h3>Prerequisites</h3>
<ul>
  <li>STAT 801 or STAT 802</li>
  <li>Knowledge of statistical programming language R</li>
</ul>


<br>
<br>
<h3>Course Description</h3>
This course will cover quantitative genetic analysis of complex trait genetics with emphasis on the use of molecular markers spanning the entire genome. We will discuss statistical methodologies for connecting phenotypes with high-dimensional genomic information to better understand polygenic traits from both prediction and inference perspectives. Topics will include genomic relatedness, linkage disequilibrium, population stratification, genomic heritability, genome-enabled prediction of complex traits, and statistical learning. We will use examples from the animal, plant, and human genetics literature. Additional topics will be briefly touched upon, including sequence data, gene expression, epigenetics, and bioinformatics. Homework assignments involve hands-on analysis of simulated and real genomic data available at public repositories. The course will use R/Bioconductor software for statistical computing tools.

<br>
<br>
<h3>Learning Objectives</h3>
After taking this course, the student will be able to:
<ul>
  <li>understand the statistical theory behind commonly used quantitative methods in genomics</li>
  <li>apply statistical methods to high-dimensional genomic data and analyze them using statistical com- puting tools</li>
  <li>critically review current literature in statistical and quantitative genetics</li>
</ul>

<br>
<br>
<h3>Texts and Reading Materials</h3>
Lecture slides will be provided on the class website. There will be no required textbook.

<br>
<br>
<h3>Tentative schedule</h3>
<ul>
  <li>1/12 (T): Overview of statistical genomics - Prediction vs. Inference [<a href="">HTML</a>]</li>
  <li>1/14 (R): Ordinary least-squares and the curse of dimensionality [<a href="">HTML</a>]</li>
  <li>1/19 (T): Linkage disequilibrium [<a href="">HTML</a>]</li>
  <li>1/21 (R): Relatedness due to expected resemblance - Additive relationships [<a href="">HTML</a>]</li>
  <li>1/26 (T): Relatedness due to expected resemblance - Non-additive relationships [<a href="">HTML</a>]</li>
  <li>1/28 (R): Relatedness due to genetic markers - Additive relationships [<a href="">HTML</a>]</li>
  <li>2/2 (T): Relatedness due to genetic markers - Non-additive relationships [<a href="">HTML</a>]</li>
  <li>2/4 (R): Population structure and cryptic relatedness [<a href="">HTML</a>]</li>
  <li>2/9 (T): Likelihood and Bayesian approaches in quantitative genetics [<a href="">HTML</a>]</li>
  <li>2/11 (R): Whole-genome regression - Penalized regression [<a href="">HTML</a>]</li>
  <li>2/16 (T): Whole-genome regression - Bayesian alphabet regression 1 [<a href="">HTML</a>]</li>
  <li>2/18 (R): Whole-genome regression - Bayesian alphabet regression 2 [<a href="">HTML</a>]</li>
  <li>2/23 (T): Whole-genome regression - Genomic BLUP 1 [<a href="">HTML</a>]</li>
  <li>2/25 (R): Whole-genome regression - Genomic BLUP 2 [<a href="">HTML</a>]</li>
  <li>3/1 (T): Introduction to whole-genome regression software</li>
  <li>3/3 (R): Whole-genome regression - Semi-parametric regression 1 [<a href="">HTML</a>]</li>
  <li>3/8 (T): Whole-genome regression - Semi-parametric regression 2 [<a href="">HTML</a>]</li>
  <li>3/10 (R): Estimation of heritability using pedigree and twin data 1 [<a href="">HTML</a>]</li>
  <li>3/15 (T): Estimation of heritability using pedigree and twin data 2 [<a href="">HTML</a>]</li>
  <li>3/17 (R): Estimation of heritability using genomic data [<a href="">HTML</a>]</li>
  <li>3/29 (T): Population stratification - Principal component analysis [<a href="">HTML</a>]</li>
  <li>3/31 (R): Population stratification - Mixed-linear-model association 1 [<a href="">HTML</a>]</li>
  <li>4/5 (T): Population stratification - Mixed-linear-model association 2 [<a href="">HTML</a>]</li>
  <li>4/7 (R): Population stratification - Genomic control [<a href="">HTML</a>]</li>
  <li>4/12 (T): Missing heritability - Related vs. unrelated individuals [<a href="">HTML</a>]</li>
  <li>4/14 (R): Deterministic equations for genome-enabled prediction [<a href="">HTML</a>]</li>
  <li>4/19 (T): Statistical genomics of disease - Liability threshold model 1 [<a href="">HTML</a>]</li>
  <li>4/21 (R): Statistical genomics of disease - Liability threshold model 2 [<a href="">HTML</a>]</li>
  <li>4/26 (T): Multiple-trait model [<a href="">HTML</a>]</li>
  <li>4/28 (R): Genotype x environment interaction [<a href="">HTML</a>]</li>
  <li>5/3 (T): Student presentations [<a href="">HTML</a>]</li>
</ul>









</body>
</html>
